src/controllers/action-queue/WyreClient.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+
+import { type EdgeAccount } from 'edge-core-js'
+
+import ENV from '../../../env'
+import { asPaymentMethodsResponse } from './cleaners'
+
+type WyreClientOptions = {
+  account: EdgeAccount
+}
+
+type WyreClient = {
+  // Properties:
+  +isAccountSetup: boolean,
+  // Methods:
+  getCryptoPaymentMethod(currencyCode: string): Promise<string | void>
+}
+
+const { baseUri } = ENV.WYRE_CLIENT_INIT
+
+/*
+This is an intermediary solution to interface with Wyre until the fiat-plugins
+are fully baked.
+*/
+export const makeWyreClient = async (opt: WyreClientOptions): Promise<WyreClient> => {
+  const { account } = opt
+  const dataStore = account.dataStore
+
+  //
+  // State:
+  //
+
+  const wyreSecret = await dataStore.getItem('co.edgesecure.wyre', 'wyreSecret').catch(_ => dataStore.getItem('co.edgesecure.wyre', 'wyreAccountId'))
+  const baseHeaders = {
+    Authorization: `Bearer ${wyreSecret}`,
+    Accept: 'application/json'
+  }
+  const isAccountSetup = !!wyreSecret
+
+  //
+  // Private methods:
+  //
+
+  const handleHttpError = (requestUri: string, responseStatus: number, responseData: string) => {
+    console.info(`Failed HTTP exchange-sell response: ${responseData}`)
+    throw new Error(`Request to ${requestUri} failed with HTTP ${responseStatus}`)
+  }
+
+  //
+  // Public
+  //
+
+  return {
+    // Properties:
+    isAccountSetup,
+
+    // Methods:
+    async getCryptoPaymentMethod(currencyCode: string): Promise<string | void> {
```

Comment:
I understand that `currencyCode` lines up with their API, and is the simplest option for now, but at some point we may need to make a table going from pluginId / tokenId to Wyre-specific currency codes.

Indeed, we already have the beginning of a table in for Wyre in GuiPlugins.js:

```js
 fixCurrencyCodes: {
      ETH: { pluginId: 'ethereum' },
      BTC: { pluginId: 'bitcoin' },
      DAI: { pluginId: 'ethereum', tokenId: '6b175474e89094c44da98b954eedeac495271d0f' },
      USDC: { pluginId: 'ethereum', tokenId: 'a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48' }
    }
```

I worry that somebody trying to swap Fantom-USDC could end up getting an Ethereum address, and therefore send money to the wrong place, if we *just* use the currency code.

Reply:
Ah yes. This WyreClient should be replaced with the upcoming FiatPlugin work from Paul. I think we may be able to get away with cutting a corner here unless it may be a real issue in the short-term.

Reply:
I think we can get away without this and fix this later in future work (`fiat` actions need more work).

Reply:
It appears we may need this after all in order to support Polygon network. I will work on some changes to this PR.

Reply:
Fixed to send to the parent currencyCode always.